packages/dao-utils/src/abi/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './auctionAbi'
 export * from './managerAbi'
+export * from './governorAbi'
```

Comment:
why do we need these exports? we just export from the base index.ts

Reply:
Stills allows you to access the abis with a simpler syntax. I've changed the import statements in the providers to reflect this.